Description: Fix the show of brightness setting in cinnamon-settings
Author: leigh123linux
Origin: git of fedora packages
---
--- a/files/usr/lib/cinnamon-settings/modules/cs_power.py
+++ b/files/usr/lib/cinnamon-settings/modules/cs_power.py
@@ -58,7 +58,7 @@
                 screen = CinnamonDesktop.RRScreen.new(Gdk.Screen.get_default())
                 outputs = CinnamonDesktop.RRScreen.list_outputs(screen)
                 for output in outputs:
-                    if (output.is_connected() and output.is_laptop() and output.get_backlight_min() >= 0 and output.get_backlight_max() > 0):
+                    if (output.is_connected() and output.is_laptop()):
                         primary_output = output
                         break
             except Exception, detail:
